src/components/HomePage/SearchArea/JobTypes.js

```diff
@@ -12,11 +12,11 @@ const JobTypes = Object.freeze([
         label: "Curricular Internship",
     },
     {
-        value: "PART_TIME",
+        value: "PART-TIME",
```

Comment:
The keys in this file are also different from the ones on "JobTypes.js" from the backend. Also, Is there a reason why this file is not in the same location as the TechOptions / FieldOptions? also I believe its name should be "JobOptions.js" just to be coherent with the others.